source/common/res/features/import-notification/main.js

```diff
@@ -22,4 +23,3 @@
     setTimeout(poll, 250);
   }
 }());
-
```

Comment:
you can just use `$(date)`

```
echo "[   INFO] Build starting at $(date)"
...
echo "[   INFO] Build finished at $(date)"
```

If you could add the quotes around it do, that'd be good.